### PR TITLE
translate to pt-PT, and wrap some long lines

### PR DESCRIPTION
**Pull request by waldyrious**
also, minor fixes to the pt-BR strings

### CODE REVIEW

File: scripts/messages.js [hunk 1]
```diff
@@ -471,92 +528,127 @@ function Messages() {
...
+                'region code (<b>6GCRMQRG+59</b>).</p>' +
                 '<h2>Do I need to apply for a plus+code?</h2>' +
-                '<p>No, plus+codes already exist for everywhere and anyone can use them for free.</p>' +
-                '<p>To get the plus+code for a place just drag the map to highlight the place you want.</p>'
+                '<p>No, plus+codes already exist for everywhere and anyone can ' +
+                'use them for free.</p>' +
+                '<p>To get the plus+code for a place just drag the map to ' +
+                'highlight the place you want.</p>'
       },
+                                                                                |
```
Comment (drinckes) [inline scripts/messages.js]: Remove this line

File: scripts/messages.js [hunk 2]
```diff
@@ -471,92 +528,127 @@ function Messages() {
...
-          'pt-BR': 'Comentários'
+          'pt-BR': 'Comentários',
+          'pt-PT': 'Comentários'
       },
       // Link to github project in main menu.
       'ui-github': {
           'en': 'View project',
           'fr': 'Voir le projet',
-          'pt-BR': 'Exibir projeto'
+          'pt-BR': 'Ver projeto'
```
Comment (drinckes) [inline scripts/messages.js]: Needs a comma

**Comment by drinckes** [comment]
Nearly there. I've got a change to refactor all the messages so if you could do this soon that would help.

**Comment by waldyrious** [comment]
Sorry about those . I hope it's correct now.

**Pull request closed without merging**